### PR TITLE
Update layout.py

### PR DESCRIPTION
**Pull request by PipKat**

### CODE REVIEW

File: pyaedt/edb_core/layout.py [hunk 1]
```diff
@@ -90,40 +89,38 @@ def update_primitives(self):
 
     @property
     def primitives(self):
-        """
+        """Retrieve all prmitives.
```
Comment (PipKat) [inline pyaedt/edb_core/layout.py:92]:
```suggestion
        """Retrieve all primitives.
```

File: pyaedt/edb_core/layout.py [hunk 2]
```diff
@@ -150,13 +147,12 @@ def circles(self):
 
     @property
     def paths(self):
-        """
+        """Retrive all paths.
```
Comment (PipKat) [inline pyaedt/edb_core/layout.py:150]:
```suggestion
        """Retrieve all paths.
```

File: pyaedt/edb_core/layout.py [hunk 3]
```diff
@@ -166,12 +162,12 @@ def paths(self):
 
     @property
     def bondwires(self):
-        """
+        """Retrive all bondwires.
```
Comment (PipKat) [inline pyaedt/edb_core/layout.py:165]:
```suggestion
        """Retrieve all bondwires.
```

File: pyaedt/edb_core/layout.py [hunk 4]
```diff
@@ -15,7 +15,7 @@
     warnings.warn('This module requires pythonnet.')
 
 class EdbLayout(object):
-    """EDB Layout class."""
+    """edbLayout class."""
```
Comment (akaszynski) [inline pyaedt/edb_core/layout.py:18]:
```suggestion
    """EdbLayout class."""
```

File: pyaedt/edb_core/layout.py [hunk 5]
```diff
@@ -59,26 +59,25 @@ def db(self):
...
         -------
         dict
-            Dictionary of Layers
+            Dictionary of layers.
         """
         return self._parent.core_stackup.stackup_layers.layers
 
     @aedt_exception_handler
     def update_primitives(self):
         """
```
Comment (akaszynski) [inline pyaedt/edb_core/layout.py:73]: Should be on the same line as the start of the docstring.

**Review by akaszynski — APPROVED** [review]
Approved, but please have those methods with the `@property` decorator changed to nouns rather than verbs.

File: pyaedt/edb_core/layout.py (hunk 5 above)
Comment (PipKat) [reply pyaedt/edb_core/layout.py:73]: @akaszynski  I don't understand what you mean by this comment.  What should be on the same line as the start of the docstring?

File: pyaedt/edb_core/layout.py [hunk 6]
```diff
@@ -59,26 +59,25 @@ def db(self):
 
     @property
     def layers(self):
-        """
+        """Retrieve layers.
```
Comment (PipKat) [inline pyaedt/edb_core/layout.py:62]:
```suggestion
        """Dictionary of layers.
```

File: pyaedt/edb_core/layout.py [hunk 7]
```diff
@@ -90,40 +89,38 @@ def update_primitives(self):
 
     @property
     def primitives(self):
-        """
+        """Retrieve all primitives.
```
Comment (PipKat) [inline pyaedt/edb_core/layout.py:92]:
```suggestion
        """List of primitives.
```

File: pyaedt/edb_core/layout.py [hunk 8]
```diff
@@ -90,40 +89,38 @@ def update_primitives(self):
...
+            List of primitives.
         """
         if not self._prims:
             self.update_primitives()
         return self._prims
 
     @property
     def polygons_by_layer(self):
-        """
+        """Retrieve polygons by layer.
```
Comment (PipKat) [inline pyaedt/edb_core/layout.py:105]:
```suggestion
        """Dictionary of polygons by layer.
```

Comment (PipKat) [inline pyaedt/edb_core/layout.py:105]:
```suggestion
        """Dictionary of primitives with layer names as keys,
```

File: pyaedt/edb_core/layout.py [hunk 9]
```diff
@@ -134,12 +131,12 @@ def rectangles(self):
 
     @property
     def circles(self):
-        """
+        """Retrieve all circles.
```
Comment (PipKat) [inline pyaedt/edb_core/layout.py:134]:
```suggestion
        """List of circles.
```

File: pyaedt/edb_core/layout.py [hunk 10]
```diff
@@ -150,13 +147,12 @@ def circles(self):
 
     @property
     def paths(self):
-        """
+        """Retrieve all paths.
```
Comment (PipKat) [inline pyaedt/edb_core/layout.py:150]:
```suggestion
        """List of paths.
```

Comment (PipKat) [inline pyaedt/edb_core/layout.py:150]:
```suggestion
        """List of paths.
```

File: pyaedt/edb_core/layout.py [hunk 11]
```diff
@@ -166,12 +162,12 @@ def paths(self):
 
     @property
     def bondwires(self):
-        """
+        """Retrieve all bondwires.
```
Comment (PipKat) [inline pyaedt/edb_core/layout.py:165]:
```suggestion
        """List of bondwires.
```

Comment (akaszynski) [reply pyaedt/edb_core/layout.py:165]: Oops, didn't see you were editing this, sorry!

**Pull request merged**